framework/model/module/module.go

```diff
@@ -328,15 +328,30 @@ func Main(bundle string, modules []Module) {
 	readyMgr := &moduleReadyManager{moduleReadyCheckers: map[string][]readyChecker{}}
 
 	// init configController if configSource field is used
-	cc, err := bootstrap.NewConfigController(config, mgr.GetConfig())
+	configController, err := bootstrap.NewConfigController(config, mgr.GetConfig())
 	if err != nil {
 		log.Errorf("start ConfigController error: %+v", err)
 		os.Exit(1)
 	}
 
+	istioConfigController, err := bootstrap.NewIstioConfigController(config)
```

Comment:
If istiod(xds source) is not ready, will it block?

Reply:
I'll optimize the procedure later